sharing_portal/migrations/0016_day_pass_fields.py

```diff
@@ -32,7 +32,7 @@ class Migration(migrations.Migration):
             field=models.ForeignKey(null=True, on_delete=models.deletion.CASCADE, related_name='reproducibility_project', to='projects.project'),
         ),
         migrations.CreateModel(
-            name='DayPassRequest',
+            name='DaypassRequest',
```

Comment:
In general historic migrations should be considered frozen. Is this pure documentation? Or, does changing the class name of the model actually also change the table name?

Reply:
@diurnalist Changing the classname in the model would change the table name, if they differed in more than just case, since Django just lower cases the model name to get the table name. So in this case, it functions as documentation, and isn't technically needed.

There is a migration operation `RenameModel`, which can be used to change table names when a model changes. However, it doesn't seem to work in this situation, since it seems the table name doesn't change.

Reply:
OK, I wasn't sure if Django did conversion from camelCase to snake_case, thought maybe it would think it was daypass versus day_pass 👍🏻